docsbox/docs/views.py

```diff
@@ -99,9 +97,7 @@ def post(self, file_id):
                 save_in_via = False
 
             elif file_id and is_valid_uuid(file_id):
-                file_info = get_file_info(request, file_id)
-
-                file_path = file_info["file_path"]
+                file_info = get_file_info(file_id, request.headers.get('Content-Disposition'), request.headers.get('Content-Type'))
```

Comment:
Do we need the non-convertable Unknown/corrupted response at this level? I think we could move that to the worker as well. That way we would never need to download the file here. It still downloads the file if there is no content type.

The worker could respond with Unknown/corrupted and non-convertable when the status is queries the next time.

Would this change cause issues in Ulsa or Haipa implementation? If so, we could add a new /v2/ api for this.